### PR TITLE
style: convert yes/no to true/false

### LINKED ISSUE #334: do not use yes/no in doc
As per the [vote](https://github.com/ansible-community/community-topics/issues/116) we should _not_ be using `yes`/`no` for truthiness in the docs.

### PR DESCRIPTION
**Pull request by hunleyd**
Convert yes/no to true/false respectively across the collection

fixed #334

### CODE REVIEW

**Comment by hunleyd** [comment]
does this look sane @Andersson007 ?

**Review by Andersson007 — APPROVED** [review]
@hunleyd thanks for the great work! Feel free to merge and backport

File: plugins/modules/postgresql_sequence.py [hunk 1]
```diff
@@ -275,7 +275,7 @@
     description: Shows if the sequence cycle or not.
     returned: always
     type: str
-    sample: 'NO'
+    sample: 'false'
```
[the remark this answers is not in the document]
Comment (hunleyd) [reply plugins/modules/postgresql_sequence.py:278]: #338

**Comment by Andersson007** [comment]
@hunleyd ah, manual backport is needed, if there's any difficulty with it, please let me know

**Pull request merged**